addons/website_slides/data/mail_data.xml

```diff
@@ -9,16 +9,18 @@
                 <div style="margin: 0px; padding: 0px;">
                     <p style="margin: 0px; padding: 0px; font-size: 13px;">
                         Hello<br/><br/>
-                        A new ${object.slide_type} <strong>${object.name}</strong> has been published on ${object.channel_id.name} at ${format_datetime(object.write_date, tz=user.tz)}
+                        There is something new in the course, <strong>${object.channel_id.name}</strong>, you are following:<br/><br/>
```

Comment:
why ',' surrounding course name ? I'm pretty sure you can remove those.

Reply:
Yes, you can!